pkg/util/validation/validation.go

```diff
@@ -105,6 +105,16 @@ func IsValidPortNum(port int) bool {
 	return 0 < port && port < 65536
 }
 
+// IsValidGroupId tests that the argument is a valid gids.
+func IsValidGroupId(gid int64) bool {
```

Comment:
We have some incompat going on.  UNIXes generally leave room for -1 and -2 UIDs to mean something, but they usually define uid_t as unsigned. If we're treating UID as signed just because Docker does - is that kosher?  It's probably not a huge issue, but it seems a bit off.